app/helpers/settings.py

```diff
@@ -63,3 +66,30 @@ def process_configuration_files(self):
                     re.compile(whitelist_val_to_check.strip(), re.IGNORECASE)
                 except Exception:
                     self.failing_regular_expressions.add(whitelist_val_to_check)
+
+        try:
+            self.print_outliers_to_console = self.config.getboolean("general", "print_outliers_to_console")
+        except NoOptionError:
+            self.print_outliers_to_console = 0
+
+        try:
+            self.es_save_results = self.config.getboolean("general", "es_save_results")
+        except NoOptionError:
+            self.es_save_results = 0
+
+        try:
+            self.list_derived_fields = self.config.items("derivedfields")
+        except NoSectionError:
+            self.list_derived_fields = dict()
+
+        try:
+            self.list_assets = self.config.items("assets")
+        except NoSectionError:
+            self.list_assets = dict()
+
+        try:
+            config.getint("terms", "terms_batch_eval_size")
+        except (NoOptionError, NoSectionError):
+            helpers.singletons.logging.logger.warning("No value has been defined for the parameter " +
```

Comment:
I'm not sure a warning is a good idea....  But I think it's important to prevent that this parameter is not set.
        
Note that this must be done for Metrics as well

Reply:
Don't use default value,  and move the terms_batch_eval_size check to additional settings of the terms model itself (same for metrics)